cmd/rest/client.go

```diff
@@ -117,5 +149,43 @@ func NewClient(url *url.URL, newCustomTransport func() *http.Transport, newAuthT
 		httpIdleConnsCloser: tr.CloseIdleConnections,
 		url:                 url,
 		newAuthToken:        newAuthToken,
+		connected:           online,
+
+		MaxErrResponseSize:  4096,
+		HealthCheckPath:     "",
```

Comment:
can we just use `/minio` as default healthCheckPath?

Reply:
We can, but I would rather have it off by default and have the user code specify it.

Right now `""` means that it is disabled by default. 

Technically we just need a response - we only care if the host:port responds to a request. We don't care about the status code returned.